run_unittests.py

```diff
@@ -124,8 +124,12 @@ def tearDown(self):
         super().tearDown()
 
     def _run(self, command):
-        self.output += subprocess.check_output(command, stderr=subprocess.STDOUT,
-                                               env=os.environ.copy())
+        try:
+            self.output += subprocess.check_output(command, stderr=subprocess.STDOUT,
+                                                env=os.environ.copy())
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->